src/supported-keyframes.js

```diff
@@ -0,0 +1,14 @@
+import prefix from './prefix'
+
+/**
+ * Test if a keyframe at-rule should be prefixed or not
+ *
+ * @param {String} vendor prefix string for the current browser.
+ * @return {Boolean}
+ * @api public
+ */
+
+export default function supportedKeyframes() {
```

Comment:
Just realized that we should pass the '@keyframes' here, because user might have prefixed it manually, so we need to check if it is already prefixed and not prefix again, right?

Reply:
In case user passes `@-webkit-keyframes something` we should accept that too, since it would be easier for jss-vendor-prefixer to just pass the key.

Reply:
I would say if the string starts with `@-` it is already prefixed

Reply:
Ok, will do it now. It makes sense to move all of prefixing logic to `css-vendor`